docs/dev/how-to/verify-contracts.md

```diff
@@ -85,7 +85,7 @@ const zkSyncTestnet =
         url: "https://testnet.era.zksync.dev",
         ethNetwork: "goerli",
```

Comment:
@idea404 also need to be sure HH configs support Sepolia. It should but good to confirm.

Reply:
support will be released for the `deploy` plugin in one hour. JS SDK now supports Sepolia network in `Provider.getDaultProvider()`